spring-cloud-dataflow-docs/src/main/asciidoc/task-developer-guide.adoc

```diff
@@ -57,118 +57,233 @@ If you have debug mode turned on, you can get even more information, in a line s
+[[task-dev-guide-creating-project]]
+=== Creating the Spring Task Project using Spring Initializer
+Now let's create and test an application that prints a timestamp to the console.
 
-[[task-dev-guide-building-timestamp-task-making-boot-task]]
-=== Making a Spring Boot Application into a Spring Cloud Task
+To do so:
 
-Now that we have a Boot Application, we can turn it into a Spring Cloud Task, to make the following changes:
+. Visit the link:https://start.spring.io/[Spring Initialzr] site.
+.. Create a new Maven project with a *Group* name of `io.spring.demo` and an *Artifact* name of `timestamp`.
+.. In the Dependencies text box, type `task` to select the `Cloud Task` dependency.
+.. In the Dependencies text box, type `jdbc` then select the `JDBC` dependency.
+.. In the Dependencies text box, type `h2` then select the `H2`. (or your favorite database)
+.. Click the *Generate Project* button
+. Unzip the timestamp.zip file and import the project into your favorite IDE.
 
-* Add a `starter-task` dependency (which makes a Spring Boot application into a Spring Cloud Task in the `pom.xml` file).
-* Add an H2 (in-memory database), because all Spring Cloud Task applications require a database.
-* Add the `@EnableTask` annotation (which makes a Spring Boot application into a Spring Cloud Task in Java).
 
-To do all that:
+[[getting-started-writing-the-code]]
+=== Writing the Code
 
-. Open the `pom.xml` file.
-. At line 34, add the following block:
-+
-[source,xml]
+To finish our application, we need to replace the generated `TimestampApplication` code with the following contents so that it will launch a Task.
+[source,java]
 ----
-<dependency>
-  <groupId>org.springframework.cloud</groupId>
-  <artifactId>spring-cloud-starter-task</artifactId>
-</dependency>
-<dependency>
-  <groupId>com.h2database</groupId>
-  <artifactId>h2</artifactId>
-</dependency>
-<dependency>
-  <groupId>org.springframework.boot</groupId>
-  <artifactId>spring-boot-starter-jdbc</artifactId>
-</dependency>
+package io.spring.demo.timestamp;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.task.configuration.EnableTask;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+@EnableTask
+public class TimestampApplication {
+
+	@Bean
+	public CommandLineRunner commandLineRunner() {
+		return new TimestampCommandLineRunner();
+	}
+
+	public static void main(String[] args) {
+		SpringApplication.run(TimestampApplication.class, args);
+	}
+
+	public static class TimestampCommandLineRunner implements CommandLineRunner {
+
+		@Override
+		public void run(String... strings) throws Exception {
+			DateFormat dateFormat = new SimpleDateFormat("YYYY-MM-dd");
```

Comment:
So, we aren't printing the time with this format. Maybe include `HH:ss` also in it, so the Application does produce the time as well?